reco_utils/recommender/deeprec/config/asvd.yaml

```diff
@@ -41,12 +41,12 @@ train:
 #metric :'auc','logloss', 'group_auc'
 info:
     show_step : 100   #print training information after a certain number of mini-batch 
-    save_model: True   #whether to save modl
+    save_model: True   #whether to save models
     save_epoch : 1    #is save_model is set to True, save the model every save_epoch.
     metrics : ['auc','logloss']  #metrics for evaluation.
     pairwise_metrics : ['mean_mrr', 'ndcg', "group_auc"]
-    MODEL_DIR : ./tests/resources/deeprec/asvd/model/din_model/
-    SUMMARIES_DIR : ./tests/resources/deeprec/asvd/summary/din_summary/
+    MODEL_DIR : ./tests/resources/deeprec/asvd/model/asvd_model/
```

Comment:
can you check this line? if at the top of the file you changed to slirec, shouldn't we change here as well slirec? Not sure if you prefer to have a more general name (like `sequential` or similar)

Reply:
The data is shared for different models, but the log and model path still need to be separated.